tests/test_http_client.py

```diff
@@ -11,13 +11,18 @@
 
 
 class StripeClientTestCase(object):
-    REQUEST_LIBRARIES = ["urlfetch", "requests", "pycurl", "urllib.request"]
+    REQUEST_LIBRARIES = [
```

Comment:
Is this change related to types?

Reply:
Yes. See the changes in `stripe/http_client.py`. `urllib.request` isn't publicly defined in the types when you `import urllib`, so instead we are doing `import urllib.request as urllibrequest`.